apps/intellij/src/main/kotlin/dev/nx/console/graph/actions/NxGraphFocusTaskAction.kt

```diff
@@ -20,6 +18,10 @@ import kotlinx.coroutines.*
 class NxGraphFocusTaskAction(private val targetDescriptor: NxTargetDescriptor? = null) :
     DumbAwareAction() {
 
+    init {
+        shortcutSet = CustomShortcutSet.fromString("control shift G")
```

Comment:
We should find a way to make this customizable via the keyboard map settings

Reply:
yeah fair enough. I can find some other predefined shortcut set. Any suggestions on the key combo?

Reply:
You can keep the same combo, but just allow it to be configured with a custom keyboard shortcut.

Reply:
done